docs/api-routes/api-middlewares.md

```diff
@@ -34,7 +34,9 @@ export const config = {
 
 The `api` object includes all configs available for API routes.
 
-`bodyParser` Enables body parsing, you can disable it if you want to consume it as a `Stream`:
+`bodyParser` Enables body parsing. If you want to consume it as a `Stream`, you can disable it.
+
+Disabling it will allow you to verify the raw body of a **webhook** request.
```

Comment:
```suggestion
One use case for disabling the automatic `bodyParsing` is to allow you to verify the raw body of a **webhook** request, for example [from GitHub](https://docs.github.com/en/developers/webhooks-and-events/webhooks/securing-your-webhooks#validating-payloads-from-github).
```
Should we link the an example doc showing verifying a webhook payload?

Reply:
@ijjk Yeah, I think linking here example is great! I'll add these changes, thanks!